src/catalog/product-scheduled-changes.md

```diff
@@ -70,5 +70,10 @@ _Scheduled change_
 
 1. On the top bar click <span class="btn">Remove from Update</span>.
 
+1. In the opened modal, select **Delete the Update** and click **Done** button
```

Comment:
For user docs, we don't use "modal". When a response is required, it is a "dialog".

```suggestion
1. In the dialog, select **Delete the Update** and click **Done**.
```